packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -245,6 +245,15 @@ export interface ISummaryRuntimeOptions {
 export interface IContainerRuntimeOptions {
     summaryOptions?: ISummaryRuntimeOptions;
     gcOptions?: IGCRuntimeOptions;
+    /**
+     * Control whether the ContainerRuntime includes AgentScheduler in its registry, whether an instance is created
+     * at _scheduler, and whether it subscribes to leadership.  This option will be removed in a future release, so it
+     * is recommended to opt-out in preparation for that change.  If you still require AgentScheduler and/or leader
+     * election, you should explicitly include AgentSchedulerFactory in the container registry, explicitly instantiate
+     * an instance of it using createRootDataStore, and explicitly register for leadership election using
+     * TaskSubscription.
+     */
+    omitAgentSchedulerAndLeaderElection?: boolean;
```

Comment:
```suggestion
    omitAgentSchedulerAndLeaderElection?: true;
```

Reply:
I expect we'll change this to be on-by-default and permit opting-out (step 3 in the plan) before removing, in which case boolean is probably what we want here?